kernel/src/main/scala/me/rexim/morganey/syntax/Language.scala

```diff
@@ -32,10 +32,18 @@ object Language {
 
   val abstractionDot     = "."
 
+  val comma              = ","
+
   val whiteSpacePattern  = """(\s|//.*|(?m)/\*(\*(?!/)|[^*])*\*/)+"""
 
   val leftParenthesis    = "("
 
   val rightParenthesis   = ")"
 
+  val leftBracket        = "["
+
+  val rightBracket       = "]"
+
+  val dotDot             = ".."
```

Comment:
Thank you! `rangeOperator` or `range` are definitely better names than `dotDot`.